### PR TITLE
App 8 - Share UIActivity

### PR DESCRIPTION
**Pull request by valadzko**
feature 1,2 Simulator Screen Recording - iPhone 11 - 2022-09-27 at 22 57 54

feature 3 Simulator Screen Recording - iPhone 11 - 2022-09-27 at 22 59 42

### CODE REVIEW

**Comment by valadzko** [comment]
https://user-images.githubusercontent.com/12141346/192622876-7457fbd8-1b21-4b88-b4bf-7101da8d90dd.mp4

File: UIKitProjects/ShareViewController.swift [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+//  UIKitProjects
+//
+//  Created by Pavel on 22.09.22.
+//
+
+import UIKit
+/// Контроллер который позволяет расшарить некоторые данные о себе в соц сети и другие приложения
+class ShareViewController: UIViewController {
+    
+    @IBOutlet weak var textFieldForSharing: UITextField!
```
Comment (AlexanderKokh) [inline UIKitProjects/ShareViewController.swift:12]:
```suggestion
    @IBOutlet weak var  sharingTextField: UITextField!
```

File: UIKitProjects/ShareViewController.swift [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    @IBOutlet weak var genderSegmentedControl: UISegmentedControl!
+    @IBOutlet weak var isITSpecialist: UISwitch!
```
Comment (AlexanderKokh) [inline UIKitProjects/ShareViewController.swift:14]:
```suggestion
    @IBOutlet weak var iTSpecialistSwitch: UISwitch!
```

File: UIKitProjects/ShareViewController.swift [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+    func captureDataAndShare() {
+        let activityViewController = UIActivityViewController(
+            activityItems: captureItemsToShare(),
+            applicationActivities: nil)
+        
+        present(activityViewController, animated: true)
+    }
+}
+/// Реализация протоколов для PickerView
```
Comment (AlexanderKokh) [inline UIKitProjects/ShareViewController.swift:69]:
```suggestion
/// Реализация протоколов для PickerView 
либо в данном случае достаточно 
/// UIPickerViewDelegate, UIPickerViewDataSource 
```

**Pull request closed without merging**